docs/standard/cross-platform/support-for-windows-store-apps-and-windows-runtime.md

```diff
@@ -10,17 +10,17 @@ helpviewer_keywords:
   - ".NET Framework, and Windows Runtime"
 ms.assetid: 6fa7d044-ae12-4c54-b8ee-50915607a565
 ---
-# .NET Framework Support for Windows Store Apps and Windows Runtime
+# .NET Framework Support for Microsoft Store Apps and Windows Runtime
```

Comment:
should this article be here? It seems like it should be the .NET Framework area.

Reply:
Agreed. Will move it.

Reply:
It was actually in the .NET Framework TOC, but in the Standard folder. I moved it so the breadcrumb reflects the TOC it's in.